xsd/siri/siri_requests-v2.0.xsd

```diff
@@ -624,7 +628,7 @@ be sent in the initial response (or a portion of the data if MoreData is set to
      <xsd:group ref="ProducerRequestEndpointGroup"/>
      <xsd:group ref="DelegatorEndpointGroup">
       <xsd:annotation>
-       <xsd:documentation>If request has been proxied by an intermediate  aggregting system, trackng informattion relating to the original requestor. This allows the aggregation to be stateless.</xsd:documentation>
+       <xsd:documentation>If request has been proxied by an intermediate aggregting system, trackng informattion relating to the original requestor. This allows the aggregation to be stateless.</xsd:documentation>
```

Comment:
"aggregting" should be "aggregating" and "trackng" should be "tracking"

Reply:
Thanks, resolved this!